RoddyCore/src/de/dkfz/roddy/config/ConfigurationValue.groovy

```diff
@@ -244,41 +255,12 @@ class ConfigurationValue implements RecursiveOverridableMapContainer.Identifiabl
      * @param dataSet
      * @return
      */
-    File toFile(Analysis analysis, DataSet dataSet) throws ConfigurationError {
-        File f = toFile(analysis)
+    File toFile(Analysis analysis, DataSet dataSet = null) throws ConfigurationError {
```

Comment:
The evaluation order here may be important and should maybe get documented.
 1. configuration itself (Not clear here from the code, what the first configuration is. Could be an analysis or context configuration as well, or not?)
 2. analysis-configuration
 3. dataset-configuration

Reply:
Sorry, I was unclear. From the code the evaluation order is clear. But it should be documented in the documentation (read the docs).